loader/include/Geode/ui/TextAreaV2.hpp

```diff
@@ -0,0 +1,241 @@
+#pragma once
+
+#include <cocos2d.h>
+#include "PaddingNode.hpp"
+
+namespace geode {
+    class GEODE_DLL TextAreaV2 : public PaddingNode {
```

Comment:
The naming convention for generic Geode UI classes is to give them short, reasonable names, ones that you would expect the given class to have in any UI framework (see `ScrollLayer`, `TextInput`, `ListView`, etc.) This should be called something analogous to `TextArea` like `MultilineLabel`, or possibly even just `Label` or `Text` (these would definitely need further discussion though)